app/assets/javascripts/appointments.js.coffee

```diff
@@ -79,6 +91,19 @@ $ ->
         url: "/multiple_procedures/reset_procedures.js"
         data: data
 
+  $(document).on 'click', '.uncomplete_visit', ->
+    appointment_id = $(this).parents('.row.appointment').data('id')
+    data = appointment: completed_date: null
+    $.ajax
+      type: 'PUT'
+      data: data
+      url: "/appointments/#{appointment_id}"
```

Comment:
I think we should always include the :format in URLs. so url: "/appointments/#{appointment_id}.js"